sky/core.py

```diff
@@ -703,8 +722,9 @@ def job_status(cluster_name: str,
     Args:
         cluster_name: (str) name of the cluster.
         job_ids: (List[str]) job ids. If None, get the status of the last job.
+        stream_logs: (bool) whether to stream logs.
     Returns:
-        Dict[Optional[int], Optional[job_lib.JobStatus]]: A mapping of job_id to
+        Dict[Optional[int], Optional[sky.JobStatus]]: A mapping of job_id to
         job statuses. The status will be None if the job does not exist.
         If job_ids is None and there is no job on the cluster, it will return
         {None: None}.
```

Comment:
Our manually annotated `(type)` can be removed

Reply:
Removed. Thanks!